gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -1833,7 +1833,7 @@ def storagePods_locked
     ensure
       @cacheLock.sync_unlock if unlock
     end
-    $vim_log.info "MiqVimInventory.dataStores_locked: loaded Datastore cache for #{@connId}"
+    $vim_log.info "MiqVimInventory.storagePods_locked: loaded Datastore cache for #{@connId}"
```

Comment:
That's much better but the whole file is like this already.  Maybe we can go through and move them all over.